src/DynamoWebServer/Messages/MessageHandler.cs

```diff
@@ -522,7 +523,27 @@ private string GetInOutPortsData(NodeModel node)
                 stringBuilder.Append("],");
             }
 
-            stringBuilder.Append("\"InPorts\": [");
+            if (node is VariableInputNode)
+            {
+                var type = node.GetType();
+                if (type.Name == "PythonNode")
+                {
+                    var script = type.GetProperty("Script").GetValue(node, null).ToString();
+
+                    stringBuilder.Append("\"Script\":\"");
+                    stringBuilder.Append(script.
+                            Replace("\n", "\\n").
+                            Replace("\"", "\\\""));
+                    stringBuilder.Append("\", ");
+                }
+
+                stringBuilder.Append("\"varInputs\": [");
+            }
+            else
+            {
+                stringBuilder.Append("\"InPorts\": [");
+            }
+
```

Comment:
Another thing, I noticed the logic around here to be:

```csharp
if (node is CodeBlockNodeModel)
{
    // Does something
}

if (node is VariableInputNode)
{
    // Does something else
}
else
{
    // CodeBlockNodeModel will get here also...
}
```

The `else` will include `CodeBlockNodeModel`, just want to make sure that is expected?